crashlytics/src/commonMain/kotlin/co/touchlab/crashkios/crashlytics/CrashlyticsCalls.kt

```diff
@@ -5,6 +5,8 @@ interface CrashlyticsCalls {
     fun sendHandledException(throwable: Throwable)
     fun sendFatalException(throwable: Throwable)
     fun setCustomValue(key: String, value: Any)
+
```

Comment:
Minor, but we should either drop the extra line, or add them between the others.